extensions/arc/src/ui/dashboards/postgres/postgresParameters.ts

```diff
@@ -607,14 +612,31 @@ export abstract class PostgresParametersPage extends DashboardPage {
 		});
 	}
 
+	private refreshParametersTable(): void {
+		this.engineSettings.map(parameter => {
+			let param = this._parameters.find(p => p.parameterName === parameter.parameterName);
```

Comment:
Should probably null check `param` in case it wasn't found in `_parameters` and is undefined.